.vscode/settings.json

```diff
@@ -37,5 +37,7 @@
             "pattern": "**/*.spec.ts",
             "command": "yarn test:theia"
         }
-    ]
-}
\ No newline at end of file
+    ],
+    "editor.tabSize": 4,
```

Comment:
It won't work: we use 2 spaces for most of JSON files and 4 spaces for others. What is the issue with using auto detect in VS code? it is enabled by default.

Reply:
We can make it explicit with: `"editor.detectIndentation": true`.

Reply:
ok, then 4 can be the default and for JSON we should override with 2, also `"editor.insertSpaces": true` should be added

Reply:
probably we need to set `"editor. detectIndentation": false` then as well and reformat existing code, mostly JSON files with 4 spaces

Reply:
I don't understand why should we set tabSize to 2 for json files, but reformat then to 4 spaces?

Reply:
I will add
```
    "[json]": {
        "editor.tabSize": 2
    }
```

but I think that we can leave detectIndentation to true.  It will do the right thing when modifying existing files that don't have an explicit rule.

Reply:
> I don't understand why should we set tabSize to 2 for json files, but reformat then to 4 spaces?

I meant reformat files with 4 spaces

Reply:
ok about `detectIndentation`